crossing-training.lic

```diff
@@ -231,13 +231,21 @@ class CrossingTraining
     return unless @settings.tithe
     delta = Time.now - (UserVars.tithe_timer || Time.now - 14_400)
     return if (delta < 4 * 60 * 60 && DRStats.paladin?) || (delta < 60 * 10 && DRStats.cleric?)
+    currency = get_data('town')[@settings.hometown]['currency']
     return unless withdraw_exact_amount?('5 silver', @settings.hometown)
 
     wait_for_script_to_complete('pay-debt')
 
     UserVars.tithe_timer = Time.now
-    walk_to(741)
-    bput('put 5 silver kronar in almsbox', 'You drop', 'But you do not', 'attend to thy own woes')
+    ## Hardcoding Shard and Crossing almsbox rooms for now, Crossing default
+    ## TODO: check almsbox yaml setting, default to Crossing
+    if @settings.hometown == 'Shard' 
```

Comment:
This seems like it should be in `base-town.yaml`. What do you think?

Reply:
I think you're probably right. I can move that over at some point today.

Reply:
I'll check for the existence of the value for the town in 'hometown', defaulting to Crossing if none is found.  At some point I'll go visit each of them to map them and capture the lich room id.